mino/minogrpc/mod_test.go

```diff
@@ -80,6 +82,23 @@ func TestMinogrpc_BadAddress_New(t *testing.T) {
 	require.Regexp(t, "^failed to bind: listen tcp 123.4.5.6:1:", err)
 }
 
+func TestMinogrpc_BadTracer_New(t *testing.T) {
+	getTracerForAddr = fake.GetTracerForAddrWithError
+
+	addr := ParseAddress("127.0.0.1", 3333)
+
+	router := tree.NewRouter(addressFac)
+
+	_, err := NewMinogrpc(addr, router)
+	require.EqualError(
+		t,
+		err,
+		fmt.Sprintf("failed to get tracer for addr 127.0.0.1:3333: %s", fake.GetError().Error()),
```

Comment:
you can use `fake.Error("failed to get tracer for addr 127.0.0.1:3333")`, which does the `fmt.Sptintf` for you.